rootfs/etc/nginx/lua/balancer/chashsubset.lua

```diff
@@ -5,6 +5,11 @@ local resty_chash = require("resty.chash")
 local util = require("util")
 local ngx_log = ngx.log
 local ngx_ERR = ngx.ERR
+local setmetatable = setmetatable
+local tostring = tostring
+local math = math
+local table = table
+local pairs = pairs
```

Comment:
I know that localizing globals are more performant with Lua interpreter, but does it really make any difference with Luajit? Luajit seems to optimize that automatically. Do you have any benchmark for this?

Reply:
I'll test it ASAP.

Reply:
Hi @ElvinEfendi I did the test. The performance difference can be ingored. But it is recommended to use local variable to avoid race conditions. WDYT?

Reply:
Thanks for testing. Yeah, that sounds good!